### PR TITLE
bin/check correct way to run golint

### PR DESCRIPTION
**Pull request by interro**

### CODE REVIEW

File: bin/check [hunk 1]
```diff
@@ -6,6 +6,6 @@
 #> bin/check
 
 bin/check_govet
-bin/check_golint
+bin/builder_run bin/check_golint
```
Comment (soffokl) [inline bin/check]: why do we need to run `golint` through the builder docker container?

Comment (tadovas) [reply bin/check]: It seems that something happened with script - I don't know. Running inside docker was invented to keep script usable on local machines :| and now it seems we need to use docker even locally :(

Comment (soffokl) [reply bin/check]: hm, it works for me locally. Could you give more details what is not working for you?

Comment (zolia) [reply bin/check]: Can you comment on whats not working without `builder_run` ? We should not increase our build time if its not needed.

Comment (interro) [reply bin/check]: for me bin/check_golint is not showing lint warnings

**Pull request closed without merging**